### ISSUE TITLE
[Feature] Per-".zdl file" save game directory

### ISSUE THREAD

**Issue by qs-arno** [qbasicer/qzdl#82]
I may not be asking this question the best way, so let me try and explain...basically, I create a separate `*.zdl` file for each PWAD I play, and that overall environment (PWAD(s), engine, IWAD, DMFLAGS, etc.) I think of as a "profile".

I would like it if each time I launched a different "profile", the save games would be unique to that "profile".

Currently, when I go to load/save, most of the options won't work because they're not related to the PWAD I'm currently playing.  Obviously that's not ZDL's fault, but it'd be nice to have them separated out.

**Comment by qs-arno** [comment]
Just noticed this is closed out; I found commit https://github.com/qbasicer/qzdl/commit/438c58ba2bd5722baea78aaf85e2e8737d33fee3 in the history; does that address this?  Like, if I put each PWAD in its own subdirectory, will the save files wind up alongside it?

I'm not currently putting each individual PWAD in its own directory, but if doing so will let me have per-ZDL-file saves then I'll definitely start doing it.  🙂 

Otherwise, if you just closed this out for housecleaning and you know you're not going to do it (or if it's solvable some other way), that's cool.

**Comment by duganchen** [comment]
Sorry. I should not have closed it as "completed".

**Comment by duganchen** [comment]
No, that PR does not implement this. What PR does, is it makes the IWAD selection dialog box open to where you've set the DOOMWADDIR environment variable.

As for this: yeah, I'm going to close this as "not planned", as I did https://github.com/qbasicer/qzdl/issues/70 which I see as related. The way to do this would be to implement *that*, and then add the -savedir flag (which I believe is gzdoom-specific) to the preset.

**Comment by duganchen** [comment]
I was reading this:

https://www.doomworld.com/forum/topic/132475-gzdoomwoof-how-do-you-separate-saves/

And this:

https://github.com/qbasicer/qzdl/blob/master/CONFFORMAT.md

And I see we've always had this. You just add your "-save" or "-savedir" argument to the "Extra command line arguments" and it will get baked into the .zdl file.
